homeassistant/components/feedreader/event.py

```diff
@@ -0,0 +1,90 @@
+"""Event entities for RSS/Atom feeds."""
+
+from __future__ import annotations
+
+import logging
+
+from feedparser import FeedParserDict
+
+from homeassistant.components.event import EventEntity
+from homeassistant.core import HomeAssistant, callback
+from homeassistant.helpers.device_registry import DeviceEntryType, DeviceInfo
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from . import FeedReaderConfigEntry
+from .const import DOMAIN, EVENT_FEEDREADER
+from .coordinator import FeedReaderCoordinator
+
+LOGGER = logging.getLogger(__name__)
+
+ATTR_CONTENT = "content"
+ATTR_LINK = "link"
+ATTR_TITLE = "title"
+
+
+async def async_setup_entry(
+    hass: HomeAssistant,
+    entry: FeedReaderConfigEntry,
+    async_add_entities: AddEntitiesCallback,
+) -> None:
+    """Set up EZVIZ cameras based on a config entry."""
```

Comment:
This seems like a copy/paste docstring? 😄

Reply:
:see_no_evil: